specification/eventhub/resource-manager/Microsoft.EventHub/stable/2024-01-01/Clusters.json

```diff
@@ -0,0 +1,581 @@
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "Event Hubs error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/namespaces": {
+      "get": {
+        "tags": [
+          "EventHubs Clusters Namespace"
+        ],
+        "operationId": "Clusters_ListNamespaces",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "List all Event Hubs Namespace IDs in an Event Hubs Dedicated Cluster.",
+        "x-ms-examples": {
+          "ListNamespacesInCluster": {
+            "$ref": "./examples/Clusters/ListNamespacesInClusterGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "List of Namespace IDs in Cluster successfully delivered.",
+            "schema": {
+              "$ref": "#/definitions/EHNamespaceIdListResult"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response detailing why this operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/triggerUpgrade": {
+      "post": {
+        "tags": [
+          "EventHubs Clusters Upgrade"
+        ],
+        "operationId": "Clusters_TriggerUpgradePost",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "Trigger pending cluster upgrade by skipping customer-set upgrade preferences",
```

Comment:
>rigger pending cluster upgrade by skipping customer-set upgrade preferences [](http://example.com/codeflow?start=25&length=75)

Oh, so the point is to start upgrading immediately instead of on the regular schedule? Makes sense. Maybe you can tweak the wording even better.